gcp/PERMISSIONS.md

```diff
@@ -0,0 +1,97 @@
+This document is meant to be a description of the accounts that can manage a GCP cluster of the GPII at some level. Each account has a limited set of particular permissions which has been assigned in order to perform the cluster tasks.
+
+Accounts and permissions
+========================
+
+### cloud-admin@raisingthefloor.org
+
+  This is a Google group of which all the operators are members. The members of this group only have the permissions needed to allow IAM changes in the projects of the organization.
+
+  1. Organization Administrator
+
+### operator@raisingthefloor.org
```

Comment:
I'm confused here - are accounts for operators, like `stepan@` meant by `operator@raisingthefloor.org`? Like this I expect to see actual `operator@` account. I would clarify this.